alb_with_ssl_no_s3logs/sg.tf

```diff
@@ -19,13 +19,12 @@ resource "aws_security_group_rule" "sg_alb_https_ingress" {
   security_group_id = "${aws_security_group.sg_alb.id}"
 }
 
-
-
 resource "aws_security_group_rule" "sg_alb_backend_egress" {
+  count                    = "${var.backend_sg_count}"
   type                     = "egress"
   from_port                = "${var.backend_https_port}"
   to_port                  = "${var.backend_https_port}"
   protocol                 = "tcp"
   security_group_id        = "${aws_security_group.sg_alb.id}"
-  source_security_group_id = "${var.backend_security_group}"
+  source_security_group_id = "${var.backend_security_group[count.index]}"
```

Comment:
I am missing the change to the variable declaration to be of type `list`. Forgot to commit something?

Reply:
oh yes! true, updated

Reply:
also the description and why don't you use the list to automatically set the count?

Reply:
@venturel not supported by Terraform.
https://github.com/hashicorp/terraform/issues/7762

Reply:
@ringods that's not true

Reply:
@minniux can you apply the change I suggested please as we have it for example here?
https://github.com/skyscrapers/terraform-rds/blob/master/rds/security_group.tf#L15